frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -11,7 +11,7 @@
       <div class="c-voting-info">
         <h5 class="has-text-weight-bold is-uppercase c-voting-info-title">{{proposal.title}}</h5>
         <p v-html="proposal.text"></p>
-        <p class="is-size-7 has-text-grey" v-if="proposal.textDetails" v-html="proposal.textDetails"></p>
+        <p class="is-size-7 has-text-text-light" v-if="proposal.textDetails" v-html="proposal.textDetails"></p>
```

Comment:
Please add a `// TODO: ` comment above line 13 to [remove the v-html](https://github.com/okTurtles/group-income-simple/pull/493/files#r234470204) attribute and replace with something else, also with a link to the issue mentioned in that link.

Reply:
I prefer to not touch in this file to avoid conflicts and fix it directly on the PR #343  . What do you think?